src/Normalizer/EntityReferenceItemNormalizer.php

```diff
@@ -106,6 +106,10 @@ public function normalize($field_item, $format = NULL, array $context = array())
       $entity = $field_item->getEntity();
       $field_uri = $this->linkManager->getRelationUri($entity->getEntityTypeId(), $entity->bundle(), $field_name, $context);
```

Comment:
I realized that now this causes an error, as `$field_name` is not defined until the line below. But also `$field_uri` is never used...

So I can either move this line down into the `foreach ($field_keys as $field_name) {` or delete the above **2** lines.